mlir/include/mlir/Dialect/OpenMP/OpenMPClauseOperands.h

```diff
@@ -189,6 +189,10 @@ struct SimdlenClauseOps {
   IntegerAttr simdlenAttr;
 };
 
+struct TargetDeviceClauseOps {
+  llvm::SmallVector<Value> isDevicePtrVars, hasDeviceAddrVars;
+};
```

Comment:
This should be split into clause-specific structures, so `TargetDeviceClauseOps` should be replaced by `IsDevicePtrClauseOps` and `HasDeviceAddrClauseOps`, each of these holding the corresponding vector of values.

Reply:
Done!